src/syscall/syscall_libc.go

```diff
@@ -1,46 +1,66 @@
-// +build darwin
+// +build darwin nintendoswitch
 
 package syscall
 
 import (
+	"errors"
 	"unsafe"
 )
 
+var (
+	notImplemented = errors.New("syscall: not implemented")
+)
+
 func Close(fd int) (err error) {
-	return ENOSYS // TODO
+	errN := libc_close(int32(fd))
```

Comment:
What are all these syscall changes for? Are they necessary for initial support?
It seems better to me to do it in a separate PR and do it properly (without all the TODOs here).

Reply:
Yes, for nintendo switch that is actually needed (it does have file I/O). Actually I'm not sure why they weren't implemented like that since the picolibc included on tinygo has that libc calls as stubs.